vendor/github.com/hashicorp/vault/sdk/version/version_base.go

```diff
@@ -9,6 +9,6 @@ var (
 	CgoEnabled bool
 
 	Version           = "1.7.0"
-	VersionPrerelease = "rc2"
+	VersionPrerelease = ""
```

Comment:
I don't think you need to change this file - the `go mod vendor` does that as well as updating the dep in go.mod and modules.txt.

Reply:
Yes, this is the change that "go mod vendor" performed as part of step 1c